pydrive2/auth.py

```diff
@@ -95,11 +95,8 @@ def _decorated(self, *args, **kwargs):
       dirty = True
     else:
       if self.access_token_expired:
-        if self.credentials.refresh_token is not None:
```

Comment:
so, the code a few lines above - why do we call `Authorize`? and how does Authorize responsible for building the credentials at all?

Reply:
if credentials are none for some reason we need to call `decoratee` like in the CheckAuth below?

it feels like it should be very similar to the CheckAuth (with some modification related to refresh token, and `service` auth type)

Reply:
good catch, it makes sense, thanks!